professional/client/WindowServer/Dockerfile

```diff
@@ -15,12 +17,12 @@ COPY resources .
 # Exposing the port 8250 at which NCache service will listen for incoming management connection requests (NCache Manager, Monitor and Tools).
 # Exposing the port 8250-8260 for management operations.
 # For more information about ports visit https://www.alachisoft.com/resources/docs/ncache/getting-started-guide-windows/open-firewall-tcp.html;
-EXPOSE 8250-8260 9800 7800-7900 8300-8400
+EXPOSE 8250-8260 9800 8300-8400
 
-# Installing the .NET 6.0 runtime.
-# .NET 6.0 can be downloaded from https://dotnet.microsoft.com/en-us/download/dotnet.
-RUN "powershell.exe Start-Process -Filepath <Filepath of .NET 6.0 from resources folder> -NoNewWindow -Wait -PassThru -ArgumentList '/s'; \
-Remove-Item <Filepath to .NET 6.0 in working directory>;"
+#downloading and installing the .NET 6.0.
+RUN curl -SL --output dotnet-runtime-win-x64.exe https://dotnetcli.azureedge.net/dotnet/Runtime/%DOTNET_VERSION%/dotnet-runtime-%DOTNET_VERSION%-win-x64.exe \
```

Comment:
Rename and use the downloaded file as 'dotnet.exe' to make it edition natural

Reply:
updated